deliverables/README.md

```diff
@@ -31,38 +40,96 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+It's easier to keep track of, edit, and debug.
 
 #### HTML
 What are classes and IDs (and how are they different)?  
+Classes and IDs are tags that can be placed on HTML content. Classes are more broad and can be used on more than one element, while IDs are specific and can be used once per page.  
+
 What are elements?  
+Elements are the parts that make up the page. For example, the body element would be the stuff included inside the <body></body> tags.  
+
 What are tags?  
+Tags mark the beginning and end of an element. 
+
 What are attributes?  
+Attributes are extra pieces of information in an element, included inside the opening tag. 
+
 What are forms?  
+Forms are an interaction point between the user and the web page or application.  
+
 What is a div?  
+A division, or section, in an HTML document. 
 
 What are selectors?  
+Selectors are things that can be edited, like a word, paragraph, link, etc. Basically anything in HTML can be a selector.
+
 What are properties?  
+Properties are the aspects being edited: font, color, font-family, font-size, height, width, position, margin, etc. 
+
 What are values?  
+Values are the specifics given to the properties. Color: red, for example. 
+
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+CSS styles are inherited from the "parent". If you definte another style later, the later one is chosen instead. 
+
 What are two CSS attributes you can change to push an element around on the page?  
+Padding and margins. 
+
 What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+1. Link the external stylesheet to HTML 
+2. Internal stylesheet written into HTML 
+3. Inline styles 
 
 ---
 ### Section 3: Git and Github  
 What is Git?  
+A git is a distributed revision control system.  
+
 What is SCM?  
+Source code management system. 
+
 What is a VCS?  
+Version control systems.  
+
 Why is Git useful for a developer?  
+It's able to handle all types of projects making it a versatile tool. 
+
 Why is Git useful for a team of developers?  
+Git is useful for a team of developers because it allows multiple developers to work together on a feature before pushing the work to origin.  
+
```

Comment:
More precisely, it allows multiple developers to work together on a project - whether they work on a feature together or not.